ClassAttendance/src/WebUI/Program.cs

```diff
@@ -21,6 +15,7 @@ public static void Main(string[] args)
                 .ConfigureWebHostDefaults(webBuilder =>
                 {
                     webBuilder.UseStartup<Startup>();
-                });
+                })
+                .UseDefaultServiceProvider(options => options.ValidateScopes = false);
```

Comment:
Can it be configured in Startup?

Reply:
It's web host configuration, so it's can't be. [More information](https://stackoverflow.com/a/36626343/12741272)

Reply:
Sorry, but the link is incorrect here)

Reply:
It's a bad way. Use scoped factory:
https://stackoverflow.com/a/50198738

Reply:
I changed DBInitializer to be non-static now, and i'm also not directly accessing UserManager, RoleManager etc. from my WebUI appilication. Instead i'm using BLL service, where i inject UserManager service and etc (DBInitializer use IdentityService, IdentityService use UserManager and etc.). Even using scope factory not helping me solve this. 
![image](https://user-images.githubusercontent.com/58309973/82731129-58e8aa00-9d0d-11ea-8eda-0d3148e450c1.png)

Reply:
I guess the approach with DBInitializer incorrect at all.
But anyway, let's leave as is for now.
In future - use another approach.